.github/workflows/test.yaml

```diff
@@ -0,0 +1,39 @@
+name: Test
+on: [push]
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        minor_version: [8, 9, 10, 11, 12]
+        os: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
maybe overkill for now honestly

Reply:
If this becomes extra work, I will kill it, but for now, my problems lie elsewhere.

Reply:
Ah, I had just seen that some of the test runs had failed and some had succeeded and thought it may be a OS dependent thing. Should have dug a bit more - sorry!